utils/constants.jsx

```diff
@@ -536,6 +539,7 @@ export const Constants = {
             sidebarHeaderTextColor: '#ffffff',
             onlineIndicator: '#06d6a0',
             awayIndicator: '#ffbc42',
+            dndIndicator: '#e51013',
```

Comment:
What colour does the theme indicator use if the user's theme doesn't contain this field since , for example, their theme was set before this field existed?

Also, does this need to be manually added to the theme settings in the UI?

Reply:
@hmhealey this I dont know 😞  I'm waiting for @asaadmahmood to take a look in this part. for sure this color is not the right one